examples/main.tf

```diff
@@ -36,6 +36,13 @@ resource "kafka_topic" "syslog" {
     "retention.ms" = "86400000"
   }
 
+  timeouts {
+    create = "8m"
+    read   = "8m"
+    update = "8m"
+    delete = "8m"
+  }
```

Comment:
Standard terraform CRUD timeouts
https://www.terraform.io/plugin/sdkv2/resources/retries-and-customizable-timeouts